typescript/ts-node-introduction/src/org/jetbrains/dukat/nodeIntroduction/introduceNodes.kt

```diff
@@ -353,13 +356,14 @@ private class LowerDeclarationsToNodes(
                         null,
                         declaration.uid,
                         null,
-                        declaration.hasDeclareModifier()
+                        declaration.hasDeclareModifier(),
+                        inDeclaredModule || declaration.hasType
```

Comment:
Let's discuss. If I'm reading this right you actually want to have this flag to be set to true for declarations - but won't it happen automatically since in declarations we always have explicit types?

Reply:
It's also my understanding that it will happen automatically, but I added this check just to be extra sure. I mean, in case if some declaration doesn't have explicit type, it would be better if we just print Any for it, than nothing at all

Reply:
But now when I think about it, it seems that we can have such cases only if something went really wrong

Reply:
After discussion decided that we still need this check